Automattic/simplenote-electron
# Make monaco rerender after linelength changes

### Fix


After changing the line length in settings Monaco does not resize.

This is probably not the right way to do it but it works. Any thoughts @dmsnell?

### Test


1. Change lineLength in settings
2. Observe Monaco editor resizes to new width
